src/Sylius/Bundle/AdminBundle/Resources/config/services/email.xml

```diff
@@ -17,11 +17,15 @@
 
         <service id="Sylius\Bundle\AdminBundle\EmailManager\OrderEmailManagerInterface" class="Sylius\Bundle\AdminBundle\EmailManager\OrderEmailManager">
             <argument type="service" id="sylius.email_sender" />
+            <deprecated package="sylius/shop-bundle" version="1.13">The "%service_id%" service is deprecated since 1.13 and will be removed in 2.0. Use "Sylius\Bundle\CoreBundle\Mailer\OrderEmailManagerInterface" instead.</deprecated>
```

Comment:
```suggestion
            <deprecated package="sylius/admin-bundle" version="1.13">The "%service_id%" service is deprecated since 1.13 and will be removed in 2.0. Use "Sylius\Bundle\CoreBundle\Mailer\OrderEmailManagerInterface" instead.</deprecated>
```